api/src/pcapi/routes/shared/price.py

```diff
@@ -0,0 +1,7 @@
+from decimal import Decimal
+
+
+def convert_to_cent(amount: Decimal | None) -> int | None:
+    if amount is None:
+        return None
+    return int(amount * 100)
```

Comment:
On pourrait peut-être faire encore plus simple :)
```suggestion
    return amount and int(amount*100)
```

Reply:
Malheureusement, mypy râle : `error: Incompatible return value type (got "Decimal | int | None", expected "int | None")  [return-value]` :( 

Ça sera plus simple de revenir à la première version (pas de moi, c'était juste une fonction déplacée)